docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>nit: experimnt is spelled wrong.</p><p> </p><p>Also, can you add a name and description to the demo tensorboard, so as demonstrate our idealized use case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The typo is fixed. </p><p>Done adding experiment name and description.</p>